llama_hub/database/README.md

```diff
@@ -31,3 +31,22 @@ documents = reader.load_data(query=query)
 ```
 
 This loader is designed to be used as a way to load data into [LlamaIndex](https://github.com/jerryjliu/gpt_index/tree/main/gpt_index) and/or subsequently used as a Tool in a [LangChain](https://github.com/hwchase17/langchain) Agent. See [here](https://github.com/emptycrown/llama-hub/tree/main) for examples.
+
+## Note
```

Comment:
Can we add a second heading referring to the Microsoft SQL Server database? will make it easier to know about what it refers about, since it's a general database readme

```md
## Notes
### Microsoft SQL Server database
```

Reply:
@EmanuelCampos done.. :)